src/components/dashboardHeader/dashboardHeader.test.tsx

```diff
@@ -1,19 +1,27 @@
 import { describe, test, expect } from 'vitest'
-import { renderWithRouter } from '../../setupTests'
+import { screen } from '@testing-library/react'
+import { renderAuthenticated } from '../../setupTests'
 import paths from '../../routing/paths'
 import DashboardHeader from './dashboardHeader'
 
 describe('<DashboardHeader />', () => {
   test('Dashboard header contains a link to the main dashboard', () => {
-    const wrapper = renderWithRouter(<DashboardHeader />)
+    const wrapper = renderAuthenticated(<DashboardHeader />)
     expect(wrapper).toBeTruthy()
 
     const a = wrapper.container.querySelector('a')
     expect(a?.href).toBe(paths.dashboard.main)
   })
 
+  test('Dashboard header contains user profile information', () => {
+    renderAuthenticated(<DashboardHeader />)
```

Comment:
I'm not going to add a test for when auth is loading because the only thing it'd meaningfully test is that user data is not displayed, but if auth is loading then there won't be a user whose info we'd display anyway.